addon/src/lib/WebApp.js

```diff
@@ -55,13 +55,6 @@ export default class WebApp {
         worker.on('detach', () => this.hub.disconnect(worker.port));
       }
     });
-    this.beacon = new PageMod({
```

Comment:
I don't know if anyone is using this or not (probably not) but we ought to check first before removing it and let them know. The whitelist has `'https://www.mozilla.org/*,about:home'`.